config/default.yaml

```diff
@@ -9,6 +9,16 @@ webserver:
   hostname: 'localhost'
   port: 9000
 
+rates_limit:
+  login:
+    window: 5 min
```

Comment:
Does `min` works? :thinking:

Reply:
Humm, it did not complain… but given it was complaining for other things I'm unsure :p